### PR TITLE
Update node version for forest service

### PR DESCRIPTION
**Pull request by ManuelStardust**

### CODE REVIEW

File: services/forest/src/service/forestLP.js [hunk 1]
```diff
@@ -66,12 +67,7 @@ module.exports = (ForestLPPersistence, restAPI) => {
           forestLPColorSet(category),
         );
       } catch (e) {
-        const error = {
-          code: 500,
-          stack: e.stack,
-          message: 'Error retrieving layer',
-        };
-        throw error;
+        throw new RestifyErrors.InternalError('Error retrieving layer');
```
Comment (erikasv) [inline services/forest/src/service/forestLP.js:70]: this is ok, but you didn't change the error on line 40

**Review by erikasv — CHANGES_REQUESTED** [review]
Could you please also change the [dependency in portfolios](https://github.com/PEM-Humboldt/biotablero-backend/blob/277c2bac85cd198b6c08810102932fd6e6e8cfdc/services/portfolios_ca/package.json#L33) to use `~` instead of `^`

**Review by cazapatamar — CHANGES_REQUESTED** [review]
Please link the issue

File: services/forest/src/service/SCIHF.js [hunk 2]
```diff
@@ -33,7 +35,9 @@ module.exports = (SCIHFPersistence, restAPI) => {
       }
 
       if (!data) {
-        throw new Error("Data for SCIHF doesn't exists in the selected area id and area type");
+        throw new RestifyErrors.InternalServerError(
```
[the remark this answers is not in the document]
Comment (ManuelStardust) [reply services/forest/src/service/SCIHF.js]: Done

File: services/forest/src/service/SCIHF.js [hunk 3]
```diff
@@ -95,7 +94,7 @@ module.exports = (SCIHFPersistence, restAPI) => {
       }
 
       if (!data) {
-        throw new Error(
+        throw new RestifyErrors.InternalServerError(
```
[the remark this answers is not in the document]
Comment (ManuelStardust) [reply services/forest/src/service/SCIHF.js]: Done

File: services/forest/src/service/SCIHF.js [hunk 4]
```diff
@@ -141,7 +140,7 @@ module.exports = (SCIHFPersistence, restAPI) => {
       }
 
       if (!data || !data.features) {
-        throw new Error(
+        throw new RestifyErrors.InternalServerError(
```
[the remark this answers is not in the document]
Comment (ManuelStardust) [reply services/forest/src/service/SCIHF.js]: Done

File: services/forest/src/service/forestLP.js (hunk 1 above)
Comment (ManuelStardust) [reply services/forest/src/service/forestLP.js:66]: Done

**Comment by ManuelStardust** [comment]
> Please link the issue
Done


> Could you please also change the [dependency in portfolios](https://github.com/PEM-Humboldt/biotablero-backend/blob/277c2bac85cd198b6c08810102932fd6e6e8cfdc/services/portfolios_ca/package.json#L33) to use `~` instead of `^`

Done

**Review by erikasv — APPROVED** [review]

**Review by cazapatamar — CHANGES_REQUESTED** [review]
It is missing to handle one error at integrity service (Line 170)

**Comment by ManuelStardust** [comment]
> It is missing to handle one error at integrity service (Line 170)

Done

**Review by cazapatamar — APPROVED** [review]

**Pull request merged**